### DISCUSSION TITLE
Unable to SSH in to the AWS AMI instance

### DISCUSSION THREAD

**Discussion by rocketsites** [getredash/redash#6814] [General]
Hoping someone can point me in the right direction here. I set up a new EC2 instance in AWS based on the Redash AMI, and associated an ssh key pair with the instance. Downloaded the private key to my local computer and attempted to connect to the instance as the ubuntu user and passing it my private key. Its saying that the key doesn't match. Im using the correct key that was associated with the instance when it was launched, so there must be something wrong about how the instance is setting up the public key for the ubuntu user. Its not an issue with my ACL, Subnet, or security poloicies, as I can ping the instance and ssh is able to communicate, its just saying the keys don't match. The permissions of my key file are set to 600, so its also not an issue with the permissions of the ssh key.

**Comment by ilsaloving** [comment]
You need to provide way more details before anyone can help you:
- What AMI did you use?  This will affect what user you're supposed to be logging in as
- what is the exact error you're getting?  ie: Is it an issue with the private key, or the host key, etc.

**Comment by rocketsites** [comment]
I was able to get in
